.circleci/config.yml

```diff
@@ -106,6 +106,17 @@ workflows:
           requires:
             - unit-tests-ui
             - unit-tests-core
+  weekly-snapshot-workflow:
+    # Run workflow every Wednesday at 12pm UTC
+    triggers:
+      - schedule:
+          cron: "0 12 * * 3"
```

Comment:
I thoughts the plan was to release on weekend, taking into account the fact that devs should merge everything before Friday night